release-please-config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "changelog-path": "CHANGELOG.md",
+  "release-type": "rust",
+  "bump-minor-pre-major": true,
+  "bump-patch-for-minor-pre-major": true,
+  "prerelease": true,
+  "draft": false,
+  "packages": {
+    "tap_core": {},
+    "tap_aggregator": {}
+  },
+  "plugins": [
+    {
+      "type": "cargo-workspace"
+    }
+  ],
```

Comment:
On the official docs this is: `"plugins": ["cargo-workspace"]` without `type`. Does `type` do something different or is this just a different way to achieve the same thing?

reference: https://github.com/googleapis/release-please/blob/bdd9b0158ce79d958da01ebf54cb6b07925bc125/docs/manifest-releaser.md?plain=1#L151

Reply:
I don't know, looks like a different way. Take a look here (same docs file): 
https://github.com/googleapis/release-please/blob/bdd9b0158ce79d958da01ebf54cb6b07925bc125/docs/manifest-releaser.md?plain=1#L547-L559

It breaks if I remove the `plugins`, so it definitely does something positive.